prepsrl/src/test/java/edu/illinois/cs/cogcomp/prepsrl/PrepSRLAnnotatorTest.java

```diff
@@ -55,4 +55,15 @@ public void testLong() throws Exception {
         View view = annotator.getView(ta);
         assertEquals("Source", view.getConstituents().get(0).getLabel());
     }
+
+    @Test
+    public void testException() throws Exception {
+        String longSentence = "That put him a stroke ahead of Curt Byrum, John Adams and " +
+                "Lonnie Clements , all of whom are looking for their first " +
+                "victory on the PGA Tour .";
+        TextAnnotation ta = TextAnnotationUtilities.createFromTokenizedString(longSentence);
+        preprocessor.annotate(ta);
+        View view = annotator.getView(ta);
+        assertEquals("ahead of", view.getConstituents().get(0).getSurfaceForm());
+    }
```

Comment:
@danyaljj as I said, I haven't been able to run the tests due to the Maven packages errors, so I don't know why that exception is thrown -- maybe 'ahead of' is not the first constituent (or it's not recognised at all?). I'll try again now and see if I can get this to work on my local machine.